tests/parsers/test_orca.py

```diff
@@ -26,3 +26,49 @@ def test_orca_default(aiida_localhost, generate_calc_job_node, generate_parser,
         'relaxed_structure': structure_attributes,
         'output_parameters': results['output_parameters'].attributes,
     })
+
+
+def test_orca_unsuccessfull(
```

Comment:
```suggestion
def test_orca_unsuccessful(
```
But maybe a better name is `failed`? This is also the language used in `aiida-core` for a job with non-zero exit status

Reply:
Thanks, I fixed the spelling, but otherwise kept it as is, to distinguish this test from from other possible failure modes in the parser.